doc/cli.md

```diff
@@ -41,12 +41,28 @@ Create SVG and PNG files from a PDF with
 
 ### Examples
 
+Create an SVG with a linked PNG for each page in a PDF.
+
 ```{code-cell} ipython3
 !svgdigitizer paginate ./files/others/example_plot_paginate.pdf
 ```
 
 Download the resulting {download}`SVG (example_plot_paginate_p0.svg)<./files/others/example_plot_paginate_p0.svg>`.
 
+In addition to the linked image, basic elements for annotating a curve in a figure
+can be embedded in the SVG with the `--template` argument.
+
+```{code-cell} ipython3
+!svgdigitizer paginate ./files/others/example_plot_paginate.pdf --template basic
```

Comment:
The help for this flag is a bit confusing. It suggests that one should provide `basic` to get a basic template. Now the flag specifically requires a filename (see below). Maybe we can set the default value to the default template in the assets folder. That would require some minor code changes in the `_create_linked_svg` helper method

Reply:
I split into template and template-file.